src/Snowcap/Emarsys/Client.php

```diff
@@ -235,6 +235,12 @@ public function getConditions()
      */
     public function createContact(array $data)
     {
+        if (isset($data['contacts']) && is_array($data['contacts'])){
+	    foreach($data['contacts'] as &$v){
```

Comment:
Can you juste rename $v to $contact to keep "context", I don't really like the use of unrecognisable variable name.